### DISCUSSION TITLE
Get headers method

### DISCUSSION THREAD

**Discussion by torgashov** [rabbitmq/rabbitmq-dotnet-client#1360] [Q&A]
### Is your feature request related to a problem? Please describe.

Hi!
Is there any way to get headers without get message body?
In my case i use dedublication plugin. In this case i can't use header route.
I have many different messages and can filter them by header by endpoings.
Thanks!

### Describe the solution you'd like

If it possible create method named BasicGetHeaders, that returns headers and deliveryTag.
If i need to skip message, i will call BasicAck.

**Comment by lukebakken** [comment]
Hello, thanks for using RabbitMQ and this client library.

RabbitMQ will always send the message body along with the headers, so this is not possible to implement.

**Reply by michaelklishin** [reply]
The only exception is an empty message that would have zero body frames. However, all frames are still delivered (pushed to the client) and deserialized together, so that does not really change anything.
